sdk/translation/ai-translation-text-rest/src/custom/customClient.ts

```diff
@@ -4,7 +4,7 @@
 import { getClient, ClientOptions } from "@azure-rest/core-client";
 import { logger } from "../logger";
 import * as coreRestPipeline from "@azure/core-rest-pipeline";
-import { TextTranslationClient } from "../clientDefinitions";
+import { TextTranslationClient } from "../clientDefinitions.js";
```

Comment:
is it a best practice for customization? I think we should put generated code into src/generated folder and try to override the generated code in src folder instead of modify generated code directly? https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/RLC-customization.md

Reply:
Moved to `generated` folder.